### ISSUE TITLE
ModuleNotFoundError: No module named 'requests'

### ISSUE THREAD

**Issue by drisspg** [bpython/bpython#988]
I am using bpython through a conda env and get the following error:
```
❯ bpython
Traceback (most recent call last):
  File "/Users/drisspg/miniconda3/envs/pytorch/bin/bpython", line 5, in <module>
    from bpython.curtsies import main
  File "/Users/drisspg/miniconda3/envs/pytorch/lib/python3.10/site-packages/bpython/curtsies.py", line 15, in <module>
    from . import args as bpargs, translations, inspection
  File "/Users/drisspg/miniconda3/envs/pytorch/lib/python3.10/site-packages/bpython/args.py", line 42, in <module>
    import requests
ModuleNotFoundError: No module named 'requests'
```

I double check 

```
❯ pip install requests
Requirement already satisfied: requests in ./miniconda3/envs/pytorch/lib/python3.10/site-packages (2.30.0)
Requirement already satisfied: urllib3<3,>=1.21.1 in ./miniconda3/envs/pytorch/lib/python3.10/site-packages (from requests) (2.0.2)
Requirement already satisfied: charset-normalizer<4,>=2 in ./miniconda3/envs/pytorch/lib/python3.10/site-packages (from requests) (3.1.0)
Requirement already satisfied: idna<4,>=2.5 in ./miniconda3/envs/pytorch/lib/python3.10/site-packages (from requests) (3.4)
Requirement already satisfied: certifi>=2017.4.17 in ./miniconda3/envs/pytorch/lib/python3.10/site-packages (from requests) (2023.5.7)
```

I am using 3.10.11

**Comment by thomasballinger** [comment]
When you start bpython, what does the banner (bpython 1.2.3 on top of ????) say?

**Comment by drisspg** [comment]
Thank you for the quick response! 

I actually recursively walked a chain of failing imports which resolved the issue. It ended up being: uninstall/re-install requests, urllib3, and then I had to install chardet. I work on the PyTorch repo and normally start by following these directions: https://github.com/pytorch/pytorch#install-dependencies when constructing a new conda env. I am not entirely sure if because of the mismatch between conda installs and pip there was something funky going on.

I will close this issue since I was able to resolve.
